server/service/integration_mdm_test.go

```diff
@@ -1981,745 +1999,6 @@ func createWindowsHostThenEnrollMDM(ds fleet.Datastore, fleetServerURL string, t
 	return host, mdmDevice
 }
 
-func (s *integrationMDMTestSuite) TestDEPProfileAssignment() {
```

Comment:
Since this file is already huuuuuge, I created a new `integration_mdm_dep_test.go` file for the DEP-specific MDM integration tests, and moved this one over there.